sdk/tests/deqp/framework/common/tcuSkipList.js

```diff
@@ -127,10 +120,28 @@ goog.scope(function() {
     tcuSkipList.getSkipStatus = function(testName) {
         var skipEntry = _skipEntries[testName];
         if (skipEntry === undefined) {
-            return { 'skip': false, 'reason': '' };
+            return this.getSkipsStatus(testName);
         } else {
             return { 'skip': true, 'reason': skipEntry };
         }
     }
 
+    /*
+     * Gets the skip status of the given tests like testpath*
+     * object with the properties "skip", a boolean, and "reason", a
+     * string.
+    */
+    tcuSkipList.getSkipsStatus = function(testName) {
```

Comment:
Please rename to an internal function (using "_" at the beginning) and to something more descriptive like "_getWildcardSkipStatus".